docs/streams/kafka_on_gdn/kafkajs-client.md

```diff
@@ -0,0 +1,72 @@
+---
+sidebar_position: 40
+title: KafkaJS Client
+---
+
+The KafkaJS client can be used to create Kafka producers, consumers. Full documentation [here](https://kafka.js.org/docs/getting-started).
+
+### Installation
+
+Install KafkaJS using:
+
+```shell
+yarn add kafkajs
+```
+
+or npm:
+
+```shell
+npm install kafkajs
+```
+
+Let's start by instantiating the KafkaJS client by pointing it towards at least one broker:
```

Comment:
```suggestion
Instantiate the KafkaJS client by pointing it towards at least one broker:
```

Reply:
I'm confused. Previous topics read like reference, this reads more like a conversational tutorial. We should keep the tone consistent.